insights-notification-schemas-java/src/main/resources/schemas/Action-out.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$comment": "This schema defines some additional information for an Action that might be useful",
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "allOf": [
+    {
+      "$ref": "./Action.json"
+    }
+  ],
+  "description": "Describes additional information fields for some of the fields of the Action",
+  "properties": {
+    "source": {
+      "additionalProperties": false,
+      "application": {
```

Comment:
This doesn't seem right, shouldn't this be inside a `properties` object?
i.e. #.properties.source.properties.application

Reply:
You're right, I don't know what I was thinking with these properties...